static/recruitment/index.html

```diff
@@ -0,0 +1,179 @@
+
+    <!--<![endif]-->
+
+    <!-- Import other components -->
+
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.css">
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400italic,400,700' rel='stylesheet' type='text/css'>
+
+    <!-- Import css -->
+    <link rel="stylesheet" href="../css/layouts/aepks.css">
+
+</head>
+<body>
+
+<!-- Nav Bar -->
+<div class="header">
+    <div class="home-menu pure-menu pure-menu-horizontal pure-menu-scrollable pure-menu-fixed">
+        <a class="pure-menu-heading" href="../index.html">AEPKS</a>
+        <ul class="pure-menu-list">
+            <li class="pure-menu-item"><a href="../fraternity/" class="pure-menu-link">Fraternity</a></li>
+            <li class="pure-menu-item"><a href="../brothers/" class="pure-menu-link">Brothers</a></li>
+            <li class="pure-menu-item"><a href="../events/" class="pure-menu-link">Events</a></li>
+            <li class="pure-menu-item"><a href="../smoker/" class="pure-menu-link">Smoker</a></li>
+            <li class="pure-menu-item"><a href="../recruitment/" class="pure-menu-link">Recruitment</a></li>
+            <li class="pure-menu-item"><a href="../contact/" class="pure-menu-link">Contact</a></li>
+        </ul>
+    </div>
+</div>
+
+<!-- Title -->
+<div class="splash-container bg-2">
+    <div class="splash">
+        <h1 class="splash-head">Recruitment</h1>
+    </div>
+</div>
+
+<!-- Main Content -->
+<div class="content-wrapper">
+
+    <!-- RUSH Event 1: Taste of the Quad Fall 2020 -->
+    <div class="content">
+        <h2 class="content-head is-center">Taste of the Quad!</h2>
+        <h4 class="event-content-subhead is-center">Saturday August 22, 2020</h4>
+        <h4 class="event-content-subhead is-center">3:00 PM</h4>
+        <h4 class="event-content-subhead is-center"><a href="">ONLINE</a></h4>
+        <div class="is-center">
+            <p class="content-head">
+            Kick off the start of rush week by meeting some of our members in an online virtual conference!
+            </p>
+        </div>
+    </div>
+
+    <!-- RUSH Event 2: Q&A FALL2020 -->
+    <div class="ribbon pure-g l-box-lrg">
+        <div class="pure-u-1">
+          <h2 class="content-head-ribbon is-center">Q and A Social Media Giveaway!</h2>
+          <h4 class="content-head-ribbon is-center">Sunday August 23, 2020</h4>
+          <h4 class="content-head-ribbon is-center">5:00 PM</h4>
+          <h4 class="content-head-ribbon is-center"><a href="https://docs.google.com/document/d/1in4xA7A-eHMzziszb6KpVL21hsdfnZGTpzb9y-UVB8Q/edit?usp=sharing">ONLINE</a></h4>
```

Comment:
Also, default hyperlink blue and purple are really hard to read on top of navy background